css/global.css

```diff
@@ -1,4 +1,6 @@
-/* button:has(div):has(div):has(div):has(div):has(svg), */
-div[data-testid="split-stays"] {
+button:has(div):has(div):has(div):has(div):has(svg[aria-label="Split stay"]),
```

Comment:
The handy [:has selector](https://css-tricks.com/the-css-has-selector/) allows you to select a parent that contains a specific child. In this case, I'm selecting a button that has a child div, that has a child div, that has a child div, that has a child svg with an aria label of "Split stay." Neat!